pallets/mips/src/lib.rs

```diff
@@ -457,11 +467,16 @@ decl_module! {
 
 impl<T: Trait> Module<T> {
     /// Retrieve all proposals that need to be closed as of block `n`.
-    pub fn proposals_maturing_at(n: T::BlockNumber) -> Vec<(MipsIndex, T::Hash)> {
+    pub fn proposals_maturing_at(block: T::BlockNumber) -> Vec<(MipsIndex, T::Hash)> {
         Self::proposal_meta()
             .into_iter()
-            .filter(|meta| meta.end == n)
-            .map(|meta| (meta.index, meta.proposal_hash))
+            .filter_map(|meta| {
```

Comment:
Previous version looks more readable (for me, it is purely personal opinion) and concise. 
It there any reason to this change that I miss?